src/hooks/useCalendar.ts

```diff
@@ -0,0 +1,29 @@
+import dayjs from 'dayjs';
+
+export default function useCalendar() {
+  const daysOfWeek = [7, 1, 2, 3, 4, 5, 6];
+
+  const getDaysInMonth = (offset: number) => {
+    const today = dayjs();
+    const desiredDay = today.add(offset, 'month');
+    const daysTemp = new Array(42);
+    const startDayOfMonth = daysOfWeek[desiredDay.startOf('month').day()]; // 1 - monday, 7 sunday
+
+    const startingDay = desiredDay.startOf('month').subtract(startDayOfMonth - 1, 'day');
```

Comment:
I imagine this magic number '1' is some kind of an offset. It also appeared before so maybe make some constant of it named _SOMETHING_OFFSET_ ?